package.json

```diff
@@ -13,8 +13,7 @@
     "lint": "eslint --ext .js --format './node_modules/eslint-friendly-formatter' .",
     "test": "mocha"
   },
-  "dependencies": {
-  },
+  "dependencies": {},
```

Comment:
You should put `eslint` into [peerDependencies](https://docs.npmjs.com/files/package.json#peerdependencies). ESLint docs [suggest](http://eslint.org/docs/developer-guide/working-with-plugins#peer-dependency) offering support from version `0.8` and onwards.